app/src/main/java/de/grobox/transportr/networks/TransportNetworks.kt

```diff
@@ -530,6 +530,19 @@ private val networks = arrayOf(
                         factory = { SpainProvider(NAVITIA) }
                     )
                 )
+            ),
+            Country(
+                R.string.np_name_hungary, flag = "🇭🇺", networks = listOf(
+                    TransportNetwork(
+                        id = NetworkId.HUNGARY,
+                        name = R.string.np_name_hungary,
+                        description = R.string.np_desc_hungary,
+                        agencies = R.string.np_desc_hungary_networks,
+                        logo = R.drawable.network_hungary_logo,
```

Comment:
Did you test this? Looks like this logo doesn't actually get added. You need to transform the SVG into an AndroidVectorDrawable, see #238

Reply:
Hello @grote 
here is the Hungary logo in AndroidVectorDrawable format:

```
<?xml version="1.0" encoding="utf-8"?>
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:viewportWidth="6"
    android:viewportHeight="3"
    android:width="64dp"
    android:height="64dp">
    <path
        android:pathData="M0 0L6 0 6 3 0 3Z"
        android:fillColor="#436f4d" />
    <path
        android:pathData="M0 0L6 0 6 2 0 2Z"
        android:fillColor="#ffffff" />
    <path
        android:pathData="M0 0L6 0 6 1 0 1Z"
        android:fillColor="#cd2a3e" />
</vector>
```

Reply:
Thanks! Would you mind adding it to your pull request?
It needs to go to `app/src/main/res/drawable/network_hungary_logo.xml`.

Reply:
Done.